src/browser/extension/options/syncOptions.js

```diff
@@ -62,6 +87,7 @@ export const isAllowed = (localOptions = options) => (
 export default function syncOptions(toAllTabs) {
   return {
     save: save(toAllTabs),
-    get: get
+    get: get,
+    subscribe: subscribe
```

Comment:
Added options subscription. This is used to re-render the `Options` component when something gets changed.